fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableClauseAnalyzer.java

```diff
@@ -906,6 +907,20 @@ public Void visitModifyColumnClause(ModifyColumnClause clause, ConnectContext co
         return null;
     }
 
+    @Override
+    public Void visitModifyColumnCommentClause(ModifyColumnCommentClause clause, ConnectContext context) {
+        if (Strings.isNullOrEmpty(clause.getColumnName())) {
+            throw new SemanticException("Column name is not set");
+        }
+
+        if (Strings.isNullOrEmpty(clause.getComment())) {
+            throw new SemanticException("Comment is not set");
```

Comment:
Can I remove/unset the comment?
If yes, `modify column a comment ""` should be OK.